src/Pages/ProductDetail/Component/ProductModal.js

```diff
@@ -0,0 +1,93 @@
+import React from "react";
+import styled, { css } from "styled-components";
+import ProductSliderIn from "./ProductSliderIn";
+
+const ProductModal = ({ isModalOpen, close }) => {
+  return (
+    <>
+      {isModalOpen ? (
+        <ModalWrap>
+          <ModalBox>
+            <ModalHeader>
+              <CountBox>1 of 10</CountBox>
+              <CloseBox onClick={close}>
+                <img src="https://24hkto1dz1v3ddyf93n0ye45-wpengine.netdna-ssl.com/wp-content/themes/kinfolk2020/assets/img/icons/svg/close.svg" />
+              </CloseBox>
+            </ModalHeader>
+            <SliderIn>
+              <ProductSliderIn />
+            </SliderIn>
+          </ModalBox>
+        </ModalWrap>
+      ) : null}
+    </>
+  );
+};
+
+export default ProductModal;
+
+const ModalWrap = styled.div`
+  ${(props) => props.theme.setSize("100%", "100%")}
+  position: fixed;
+  top: 0;
+  left: 0;
+  dispaly: block;
```

Comment:
```suggestion
  display: block;
```
그런데 이미 div가 block이군요?